### PR TITLE
PXC-docs-added-Quickstarts-and-updated-file

### PR DESCRIPTION
**Pull request by patrickbirch**

### CODE REVIEW

File: source/Quickstart-GKE.rst [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+B. As an alternative, you can run a gcloud command in the `Cloud
+Shell <https://cloud.google.com/shell/docs/quickstart>`__.
+
+**Note** *You must edit the following command and other command line
+statements to replace the ``<project name>`` placeholder with your
+project name.*
+
+::
+
+   gcloud beta container --project <project name> clusters create "standard-cluster-1" --zone "us-central1-a" --username "admin" --cluster-version "1.11.7-gke.12" --machine-type "n1-standard-4" --image-type "COS" --disk-type "pd-standard" --disk-size "100" --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "3" --enable-cloud-logging --enable-cloud-monitoring --no-enable-ip-alias --network "projects/<project name>/global/networks/default" --subnetwork "projects/<project name/regions/us-central1/subnetworks/default" --addons HorizontalPodAutoscaling,HttpLoadBalancing --enable-autoupgrade --enable-autorepair
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: keep it simple
```suggestion
  gcloud container clusters create standard-cluster-1 --project <project name> --zone us-central1-a --cluster-version 1.12 --machine-type n1-standard-4 --num-nodes=3
```

File: source/Quickstart-GKE.rst [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+In the Google Cloud console, select your cluster and then select
+``Connect``. The connect statement configures command line access. After
+you have edited the statement, you may run the command in your local
+shell or in the Cloud Shell.
+
+For this document, we are using the local shell.
+
+::
+
+   gcloud container clusters get-credentials standard-cluster-1 --zone us-central1-a --project <project name>
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
   gcloud container clusters get-credentials standard-cluster-1 --zone us-central1-a --project <project name>
```

File: source/Quickstart-GKE.rst [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+
+5. You use Cloud Identity and Access Management (Cloud IAM) to control
+   access to the cluster. You must have the ability to create Roles and
+   RoleBindings. In the following command, you must replace the email
+   address placeholder with your email address.
+
+.. code:: bash
+
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole cluster-admin --user email.address
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
   --clusterrole=cluster-admin --user=$(gcloud config get-value core/account)
```

File: source/Quickstart-GKE.rst [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+   kubectl config set-context $(kubectl config current-context) --namespace=<namespace name>
+
+A successful return statement prints the project name, location (for
+this example, a zone), and cluster name.
+
+.. code:: bash
+
+   Context "<project name>_<zone location>_<cluster name>" modified.
+
+5. You use Cloud Identity and Access Management (Cloud IAM) to control
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: move this step immediately after GKE start

File: source/Quickstart-GKE.rst [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+   kubectl create clusterrolebinding cluster-admin-binding \
+   --clusterrole cluster-admin --user email.address
+
+The return statement confirms the creation.
+
+.. code:: bash
+
+   clusterrolebinding.rbac.authorization.k8s.io/cluster-admin-binding created
+
+6. The role-based access control (RBAC) for the cluster should be
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: for GKE CRD, RBAC, Operator steps can be simplified to one command
```
kubectl apply -f deploy/bundle.yaml
```

So, in result installation will can look like
```
gcloud container clusters create ...
kubectl create clusterrolebinding cluster-admin-binding ...
kubectl create namespace ...
kubectl config set-context ...
git clone ...
kubectl apply -f deploy/bundle.yaml
kubectl apply -f deploy/secrets.yaml
kubectl apply -f deploy/cr.yaml
```

File: source/Quickstart-GKE.rst [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+
+   kubectl create namespace <namespace name>
+
+The following confirmation is returned:
+
+::
+
+       namespace/<namespace name> created
+
+4. Set the context for the namespace, which saves the namespace for
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: I think "create a new namespace and switch into it" is one step with two commands

File: source/Quickstart-GKE.rst [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+creation:
+
+.. code:: bash
+
+   perconaxtradbcluster.pxc.percona.com/cluster1 created
+
+Verifying the cluster operator
+------------------------------
+
+To verify the cluster creation, run the following command:
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
It is needed at least 10 minutes to get cluster started.
To verify the cluster creation, run the following command:
```

File: source/Quickstart-GKE.rst [hunk 8]
```diff
@@ -0,0 +1,326 @@
...
+   kubectl describe pod cluster1-pxc-0
+
+Review the detailed information for ``Warning`` statements and then
+correct the configuration. An example of a warning is as follows:
+
+::
+
+   Warning  FailedScheduling  68s (x4 over 2m22s)  default-scheduler  0/1 nodes are available: 1 node(s) didn't match pod affinity/anti-affinity, 1 node(s) didn't satisfy existing pods anti-affinity rules.
+
+Removing the cluster
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
Removing the GKE cluster
```

File: source/Quickstart-GKE.rst (hunk 6 above)
Comment (patrickbirch) [reply source/Quickstart-GKE.rst]: I don't see that sentence.

Comment (delgod) [reply source/Quickstart-GKE.rst]: I am about joining step 3 and step 4 into one step

Comment (patrickbirch) [reply source/Quickstart-GKE.rst]: Done.

File: source/Quickstart-GKE.rst [hunk 9]
```diff
@@ -0,0 +1,326 @@
...
+\* ```gcloud`` <https://cloud.google.com/sdk/docs/quickstarts>`__ The
+Google Cloud SDK is the Cloud Platform toolset. \*
+```kubectl`` <https://cloud.google.com/kubernetes-engine/docs/quickstart#choosing_a_shell>`__
+the Kubernetes command-line tool. You can use the tool to manage and
+deploy applications. Run the following command:
+
+.. code:: bash
+
+   gcloud components install kubectl
+
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
gcloud auth login

```

File: source/Quickstart-GKE.rst [hunk 10]
```diff
@@ -0,0 +1,326 @@
...
+-----------------------
+
+1. Use the ``git clone`` command to download the correct branch of the
+   percona-xtradb-cluster-operator repository.
+   **Note:** *In the git statement, you must specify the correct branch
+   with the -b option.*
+
+.. code:: bash
+
+       git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
```
Comment (delgod) [inline source/Quickstart-GKE.rst]:
```suggestion
       git clone -b release-1.0.0 https://github.com/percona/percona-xtradb-cluster-operator
```

File: source/Quickstart-GKE.rst (hunk 6 above)
Comment (delgod) [reply source/Quickstart-GKE.rst]: where it is done?

File: source/Quickstart-GKE.rst [hunk 11]
```diff
@@ -0,0 +1,326 @@
...
+this example, a zone), and cluster name.
+
+.. code:: bash
+
+   Context "<project name>_<zone location>_<cluster name>" modified.
+
+5. You use Cloud Identity and Access Management (Cloud IAM) to control
+   access to the cluster. You must have the ability to create Roles and
+   RoleBindings. In the following command, you must replace the email
+   address placeholder with your email address.
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: we don't have placeholder anymore

File: source/Quickstart-GKE.rst [hunk 12]
```diff
@@ -0,0 +1,326 @@
...
+.. code:: bash
+
+   deployment.apps/percona-xtradb-cluster-operator created
+
+8. The data section of the deploy/secrets.yaml file contains
+   base64-encoded logins and passwords for the user accounts.
+
+.. code:: bash
+
+   kubectl apply -f deploy/secrets.yaml
```
Comment (delgod) [inline source/Quickstart-GKE.rst]: also, step needed for ssl secrets

File: source/Quickstart-Minikube.rst [hunk 13]
```diff
@@ -0,0 +1,335 @@
...
+---------------------------------
+
+| Use the ``git clone`` command to download the correct branch of the
+  **percona-xtradb-cluster-operator** repository.
+| **Note:** *In the git statement, you must specify the correct branch
+  with the -b option.*
+
+.. code:: bash
+
+   $ git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
```
Comment (delgod) [inline source/Quickstart-Minikube.rst]:
```suggestion
   $ git clone -b release-1.0.0 https://github.com/percona/percona-xtradb-cluster-operator
```

File: source/Quickstart-Minikube.rst [hunk 14]
```diff
@@ -0,0 +1,335 @@
...
+   deployment.apps/percona-xtradb-cluster-operator created
+
+7. Store and manage sensitive information, such as passwords or ssh
+   keys, in the secrets.yaml.
+
+.. code:: bash
+
+   $ kubectl apply -f deploy/secrets.yaml
+
+The return statement confirms the action.
```
Comment (delgod) [inline source/Quickstart-Minikube.rst]: add ssl secrets step

File: source/Quickstart-Minikube.rst [hunk 15]
```diff
@@ -0,0 +1,335 @@
...
+
+::
+
+   perconaxtradbcluster.pxc.percona.com/cluster1 created
+
+Connecting to the Cluster
+-------------------------
+
+After a few minutes, to allow for the cluster to be updated, use the
```
Comment (delgod) [inline source/Quickstart-Minikube.rst]:
```suggestion
After 10 minutes, to allow for the cluster to be updated, use the
```

**Pull request closed without merging**